widget/storybook/src/containers/ConnectWalletsModal/mock.ts

```diff
@@ -0,0 +1,120 @@
+import type { WalletInfo } from '@rango-dev/ui';
+
+import { WalletState } from '@rango-dev/ui';
+import { WalletTypes } from '@rango-dev/wallets-shared';
+
+export const walletsInfo: WalletInfo[] = [
+  {
+    image: 'https://app.rango.exchange/wallets/coinbase.svg',
```

Comment:
It's not related to this PR but this Url doesn't work anymore.
```suggestion
    image: 'https://raw.githubusercontent.com/rango-exchange/assets/main/wallets/coinbase/icon.svg,
```